Update nginx.conf.sample to remove PHP configuration settings

gordonknoppe opened the pull request:
Increase PHP memory_limit in nginx.conf.sample to match official documentation.

Update: See conversation below, PR updated to remove PHP configurations entirely

gordonknoppe committed:
Update nginx.conf.sample memory_limit=768M

Increase PHP memory_limit in nginx.conf.sample to match official documentation.

samm-git commented:
I would personally recommend to not add **any** of the php settings in the nginx.conf. Much better place should be PHP-FPM pool configuration. Probably we can put sample text for it to the git or/and documentation. Reason is very simple - every fastcgi_param adds addtitional traffic in the nginx <-> php-fpm dialog for every request. With settings in the php.ini or php-fpm pool this is not an issue.

gordonknoppe committed:
Update nginx.conf.sample

Remove PHP configuration settings from the nginx config.  These settings belong in php.ini or php-fpm pool settings.

gordonknoppe commented:
Agreed, @magenx and @samm-git, PR updated.

Vinai commented:
Thank you for the PR! We are currently waiting for the travis builds to work again, which should happen in a few days. Once that is running and the tests are green we can continue to process your contribution.

vkublytskyi commented:
@gordonknoppe, thank you for your contribution. But I'll close this PR as we have [another one](https://github.com/magento/magento2/pull/2565) which provides more full Ngingx configuration
